test/lib/dates.lib.test.js

```diff
@@ -138,4 +138,18 @@ describe('Dates lib', () => {
       expect(result).to.be.false()
     })
   })
+
+  describe('earliestDate', () => {
```

Comment:
This needs to be the first section in the file, to match the alphabetical ordering, and the functions position in the source file.